assets/style.css

```diff
@@ -0,0 +1,600 @@
+  padding: 0;
+  -webkit-text-stroke: #cf6020;
+}
+
+.btn-nav {
+  border: 4px solid var(--color-secondary);
+  color: var(--color-secondary);
+  background: transparent;
+  padding: 0.7rem 1rem;
+  outline: none;
+  cursor: pointer;
+  transition: all 0.5s ease-in-out;
+}
+
+.btn-nav:hover {
+  background-color: #1e4bb4;
+  color: #faf9f9;
+}
+
+/* Speakers Section */
+.speaker-img {
+  width: 100%;
+  z-index: 1;
+}
+
+.main-logo {
+  width: 70px;
+  height: 70px;
+  position: absolute;
+  top: -15px;
+  left: -15px;
+  z-index: -1;
+}
+
+.subsection {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 40px;
+  justify-content: center;
+  align-items: center;
+  width: 80%;
+  margin: 30px auto;
+}
+
+.subsection-items {
+  display: grid;
+  grid-template-columns: 0.5fr 1.5fr;
+  gap: 15px;
+}
+
+.subsection-img {
+  position: relative;
+}
+
+.speaker-header {
+  text-align: center;
+  margin: 3rem 0;
+}
+
+.subsection-label {
+  color: #ec240d;
+  font-style: italic;
+}
+
+.speaker-description p {
+  margin: 1rem 0;
+}
+
+.fashion-img {
+  width: 300px;
+  height: 200px;
+  margin: 2rem 1rem;
+}
+
+.images {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2.5rem;
+  padding-bottom: 4rem;
+}
+
+.fashion-images {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 3rem;
+  position: relative;
+}
+
+.blend {
+  background: #e53f2d95;
+  width: 300px;
+  height: 200px;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  row-gap: 0.3rem;
+  top: 9%;
+  left: 3%;
+}
+
+.blend h2 {
+  color: var(--body-color);
+  font-size: 3.5rem;
+}
+
+.blend p {
+  color: var(--body-color);
+  font-size: 1.2rem;
+  font-weight: 600;
+  margin: 1.5rem 0 1.5rem 0;
+}
+
+/* Main-Program */
+.main-program {
+  background: url("../assets/images/program.svg"), #272a32;
+  width: 100%;
+  text-align: center;
+  padding: 3rem 0;
+  height: 70vh;
+}
+
+.program-labels {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 3rem;
+  gap: 1rem;
+}
+
+.program-items {
+  background: #404144b4;
+  width: 150px;
+  height: 200px;
+}
+
+.program-heading {
+  text-align: center;
+  color: var(--white);
+}
+
+.fa-artstation,
+.fa-wpforms,
+.fa-house-laptop,
+.fa-chalkboard-user {
+  font-size: 2.5rem;
+  color: var(--white);
+  margin-top: 2rem;
+}
+
+.tag {
+  color: #ec240d;
+  font-size: 1rem;
+  font-family: var(--font-headings);
+}
+
+.description {
+  color: var(--white);
+}
+
+.program-heading2 {
+  color: var(--white);
+  text-decoration: underline;
+  text-decoration-color: var(--white);
+}
+
+.program-btn {
+  display: none;
+}
+
+/* Mobile-Version */
+@media screen and (max-width: 768px) {
+  .nav-section {
+    display: none;
+  }
+
+  .hero-section {
+    max-width: 110%;
+    height: 600px;
+    margin: 0 auto;
+    justify-content: center;
+    padding: 20px;
+  }
```

Comment:
- [ ] I have noticed that your `.hero-section` is overlaping the `.main-program` for Mobile version. Consider increasing the height for the `.hero-section` class. This will help Fix the overlapping issue
 
![image](https://user-images.githubusercontent.com/52143815/175437204-ed10eb95-a335-4d71-a8f1-9a4f56d84b6a.png)